src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -23,6 +23,7 @@
 from ..timeseries._grid_frequency import GridFrequency
```

Comment:
@frequenz-floss/python-sdk-team this naming `GridFrequency` is confusing to me as it somehow clashes with the grid connection `Grid`. Do you think the name is right or is it just me? I think a name around `MicrogridFrequency`  or `ComponentFrequency` would be more suitable. If we need to change it should be done in a separate PR. Please let me know your thoughts, thanks!

Reply:
I agree, `MicrogridFrequency` sounds like a better fit, based on our earlier discussions, by maybe even just `ACFrequency`, because there's only one value for one microgrid, whether it is connected to the grid or not.

Reply:
I would call it `MicrogridFrequency` for more context, `ACFrequency` sounds too vague, frequency where? Is it just a metric?

Reply:
Yes, it is just a metric

Reply:
I created the ticket https://github.com/frequenz-floss/frequenz-sdk-python/issues/770 to address this suggestion

Reply:
or It could also be moved to `microgrid.grid`

Reply:
I don't think it is the best idea, because islands still have a frequency, and even if we will have a `grid` component for islands, it is weird to talk about a frequency for something where there is no current flowing.

Reply:
But I think the same would apply for the metric `current()`. I understood we are now defining `Grid` as an abstract component regardless if there is a grid connection or not

Reply:
I would have liked that, but is different, because when there's no grid connection, we report 0 power and current values from `grid.current()`, etc,  but not so with frequency.  `microgrid.frequency()` feels like the right place for it.

Reply:
Got it, then I'll keep the ticket with the initial suggestion just to rename it. Thanks for the clarification!